Ports/pango/package.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='pango'
+# pango 1.55 requires fontconfig 2.15 but SerenityOS uses fontconfig 2.14
+version='1.54.0'
```

Comment:
Why don't we upgrade `fontconfig` then? Although, it sounds quite unlikely in general that something would have a hard dependency on such a new fontconfig version.

Reply:
> Why don't we upgrade fontconfig then?

I don't have time to work on it, and version 1.54 is still recent enough to build GTK 3.24 and GTK 4.16.

Should I add a TODO comment like `# TODO: Upgrade fontconfig to 2.15 or later, as it's required to build pango 1.55`?

> Although, it sounds quite unlikely in general that something would have a hard dependency on such a new fontconfig version.

Here's a commit that made fontconfig 2.15 a requirement for pango.

https://gitlab.gnome.org/GNOME/pango/-/merge_requests/777/diffs?commit_id=fe7cc1d4cd57ee188e4676b65bbef367b1201118

GTK doesn't seem particularly concerned with compatibility across subprojects. In fact, the latest version of GTK4 requires pango 1.56, which was released only three months ago.